src/cargo/util/config/mod.rs

```diff
@@ -1200,6 +1200,8 @@ impl Config {
                 path.display()
             );
         }
+        tracing::event!(tracing::Level::DEBUG, ?path, ?why_load, includes, "load config from file");
```

Comment:
nit: can't we do `tracing::debug!(` and skip referencing the constant?

Reply:
Nice catch!